activestorage-openstack.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
-  s.add_dependency "fog-openstack", '~> 0.3'
+  s.add_dependency "fog-openstack", '< 2.0'
```

Comment:
I'd rather apply semantic versioning, and releasing a new version of the gem if `fog-openstack`